src/components/dashboard/Dashboard.js

```diff
@@ -289,25 +298,29 @@ const Dashboard = props => {
   // So we need to calculate the center of the screen within dynamically changed balance block width.
   const balanceHasBeenCentered = useRef(false)
 
-  const saveBalanceBlockWidth = event => {
-    if (balanceHasBeenCentered.current) {
-      return
-    }
-    const width = _get(event, 'nativeEvent.layout.width')
+  const saveBalanceBlockWidth = useCallback(
+    event => {
+      if (balanceHasBeenCentered.current) {
+        return
+      }
 
-    setBalanceBlockWidth(width)
+      const width = _get(event, 'nativeEvent.layout.width')
```

Comment:
you could just do:

`const { width } = nativeEvent.layout;`

Reply:
I will take it into account next time

Reply:
ok